docs/core/preview3/tools/dotnet-sln.md

```diff
@@ -0,0 +1,79 @@
+---
+title: dotnet-sln command | Microsoft Docs
+description: The dotnet-sln command provides a convenient option to add, remove, and list projects in a solution file.
+keywords: dotnet-run, CLI, CLI command, .NET Core
+author: spboyer
+ms.author: mairaw
+ms.date: 02/06/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: e5a72d3e-c14b-4b0a-a978-c5e54a0988c6
+---
+# Name
+
+## dotnet-sln (Tooling RC4)
+
+dotnet-sln -- Modifies a .NET Core solution file.
+
+## Synopsis
+
+```bash
+dotnet sln [<SLN_NAME>] add <project> <project>
+dotnet sln [<SLN_NAME>] add **/**
```

Comment:
the `dotnet add <project> <project>` interesting find however.

In powershell

```powershell
dir **\**\*.fsproj


    Directory: /Users/shayneboyer/projects/fsharp/FSNEtCore/src/App


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
------           2/7/17   1:51 PM            873 App.fsproj


    Directory: /Users/shayneboyer/projects/fsharp/FSNEtCore/src/Library


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
------           2/7/17   1:52 PM            848 Library.fsproj
```

returns the correct items, but 

```powershell
PS /Users/shayneboyer/projects/fsharp/FSNEtCore> dotnet sln add **\**\*.fsproj
Project `**\**\*.fsproj` does not exist.
```

Errors out when adding.